src/main/kotlin/io/bazel/kotlin/builder/tasks/jvm/KotlinJvmTaskExecutor.kt

```diff
@@ -145,6 +145,9 @@ class KotlinJvmTaskExecutor
           if (outputs.generatedKspSrcJar.isNotEmpty()) {
             context.execute("creating KSP generated src jar", ::createGeneratedKspKotlinSrcJar)
           }
+          if (outputs.generatedKspClassesJar.isNotEmpty()) {
+            context.execute("creating KSP generated classes jar", ::createdGeneratedKspClassesJar)
```

Comment:
Would you be able to add some tests for this to `KotlinJvmKspAssertionTest`?

Reply:
I wasn't able to add tests with Micronaut because that'd require upgrading the repo to Java 17 (which introduces other breakages). I'm not aware of other KSP processors that generate bytecode on the fly, so I wrote an example/dummy processor that generates class files and then added test cases to validate those class files are present in the final jar.